BarGauge: small demo description fixes (#27064)
DevExpress/DevExtreme: apps/demos/Demos/Gauges/Palette/description.md

> In this demo, the BarGauge uses one of the predefined palettes for coloring bars. To specify your own palette, assign an array of required colors to the [palette](/Documentation/ApiReference/UI_Components/dxBarGauge/Configuration/#palette) property.

Reviewer: ```suggestion
In this demo, the BarGauge uses one of the predefined palettes for coloring bars. To specify a custom palette, assign an array of colors to the [palette](/Documentation/ApiReference/UI_Components/dxBarGauge/Configuration/#palette) property.
```
